packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -126,13 +130,10 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
       }
     });
     addAutoDisposeListener(preferences.inspector.customPubRootDirectories, () {
-      _refreshInspector();
-    });
-    addAutoDisposeListener(serviceManager.isolateManager.mainIsolate, () {
-      _refreshInspector();
-      preferences.inspector.loadCustomPubRootDirectories();
+      if (serviceManager.hasConnection) {
+        _customPubRootDirectoriesRefreshDebouncer(() => _refreshInspector());
```

Comment:
can we just `await Future.delayed(...)` if we want to call this after a delay, instead of creating a Debouncer class?